bind_uploader.go

```diff
@@ -118,7 +118,7 @@ func (bu *bindUploader) createStageIfNeeded() error {
 		return (&SnowflakeError{
 			Number:   code,
 			SQLState: data.Data.SQLState,
-			Message:  err.Error(),
+			Message:  fmt.Sprint(err),
```

Comment:
Any comment why this is needed?

Reply:
Added a comment because I noticed err will not ever be addressed since there's a bunch of short circuits on err beforehand.